OSPSuite.Presentation.Importer/Core/DataSourceFileReaders/CsvDataSourceFile.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using OSPSuite.Core.Services;
+using OSPSuite.Infrastructure.Import.Services;
+
+
+namespace OSPSuite.Presentation.Importer.Core.DataSourceFileReaders
+{
+   public interface ICsvDataSourceFile : IDataSourceFile { }
+
+   public class CsvDataSourceFile : DataSourceFile, ICsvDataSourceFile
+   {
+      public CsvDataSourceFile(IImportLogger logger) : base(logger) { }
+      protected override Dictionary<string, IDataSheet> LoadFromFile(string path)
```

Comment:
Unless there is a reason for using Dictionary, use Cache

Reply:
Changed to Cache. We should actually use Cache instead of Dictionary everywhere right? We are using an IReadOnlyDictionary at some point that I am also currently thinking of changing to Cache